e2e/wdio/core/tests/button.e2e-spec.ts

```diff
@@ -0,0 +1,309 @@
+import { ButtonPo } from '../pages/button.po';
+import {
+    getAttributeByName,
+    getElementArrayLength,
+    isElementClickable,
+    scrollIntoView,
+    setValue,
+    click,
+    sendKeys,
+    checkElementScreenshot,
+    mouseHoverElement, saveElementScreenshot, addIsActiveClass
+} from '../../driver/wdio';
+import {
+    text, fdTypeOptions, iconOptions, button
+} from '../fixtures/appData/button-contents';
+import {
+    disableStateButtonsExample, disableStateButtonsHoverState,
+    iconButtonsActiveState,
+    iconButtonsExample, iconButtonsFocusState, iconButtonsHoverState,
+    menuButtonsActiveState,
+    menuButtonsExample,
+    menuButtonsFocusState,
+    menuButtonsHoverState, playgroundButtonExample,
+    sizeButtonsActiveState,
+    sizeButtonsExample, sizeButtonsFocusState,
+    sizeButtonsHoverState, stateButtonActiveState, stateButtonExample, stateButtonFocusState, stateButtonHoverState,
+    typeButtonsActiveState,
+    typeButtonsExample,
+    typeButtonsFocusState,
+    typeButtonsHoverState
+} from '../fixtures/testData/button-tags';
+
+describe('Button test suite:', function() {
+    const buttonPage = new ButtonPo();
+    const {
+        typeButtons, menuButtons, sizeButtons, iconButtons, stateButton, disableStateButtons, playgroundButton, inputLabel,
```

Comment:
When tests gets bigger one can get easily lost and you forget to copy text that describe the text. Are you keep testing click-ability of buttons but from line 68 you dont use this clickable but still you are testing it. 

My Suggestion would be to introduce nested describe

Describe"button"

    describe( buttons needs to be clickable )
        it 
        it 
        it where you dont need to repeat clickable word
    
    describe( some others)
        it 
        it 
        it where you dont need to repeat clickable word
        
 Because if I read e.g. line 127, its not really clear what you are trying to verify unless I did into the code. And the whole idea of described => it is that when it fails we know why it failed.